easybuild/easyconfigs/c/CodingQuarry/CodingQuarry-2.0-GCC-11.2.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'CodingQuarry'
+version = '2.0'
+
+homepage = 'https://sourceforge.net/p/codingquarry'
+description = "Highly accurate hidden Markov model gene prediction in fungal genomes using RNA-seq transcripts"
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'openmp': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['CodingQuarry_v%(version)s.tar.gz']
+checksums = ['1198afbf7cebcf0975c5b20d92b7a2dd6d956072fcde6e86fdce6aeae4842504']
+
+buildopts = 'CFLAGS="$CFLAGS"'
+
+files_to_copy = [
+    (['CodingQuarry', 'CufflinksGTF_to_CodingQuarryGFF3.py'], 'bin'),
```

Comment:
A python file copied to bin and no Python dependency? There are more python scripts in `QuarryFiles/scripts` and all have issues:
* The Python scripts have `#!/usr/bin/python` as a first first line.
* Requires Python2 or some alterations: `print 'run: \n\npython CufflinksGTF_to_CodingQuarryGFF3.py transcripts.gtf > out.gff3\n'`
* BioPython dependency needed: `from Bio import SeqIO`

Reply:
fixed in 7f91101 + 7a983ccad1